src/plugins/intel_cpu/src/utils/ngraph_utils.hpp

```diff
@@ -47,5 +49,24 @@ inline bool isDynamicNgraphNode(const std::shared_ptr<const ngraph::Node>& op) {
     return ret;
 }
 
+inline std::string get_port_name(const ov::Output<const ov::Node>& port, const bool is_legacy_api) {
+    std::string name;
+    // TODO: Should use tensor name as the port name, but many legacy tests still use legacy name
+    // plus sometimes it will get empty tensor name.
+    if (!is_legacy_api) {
+        name = {};
+    }
```

Comment:
Looks unnecessary, `name` is instanced with the default constructor, thus it's already empty.

Reply:
ok, will remove it.